etc/opensc.conf.in

```diff
@@ -352,6 +352,22 @@ app default {
 		secure_messaging = local_amos_eid;
 	}
 
+	# SmartCard-HSM with contact-based interface or USB-Stick
+	card_atr 3B:FE:18:00:00:81:31:FE:45:80:31:81:54:48:53:4D:31:73:80:21:40:81:07:FA {
+                driver = "sc-hsm";
+		md_read_only = false;
```

Comment:
Hmm, those md* configuration options aren't documented anywhere. Since you seem to know what you are doing, could you add some comments in opensc.conf.in?